mesonbuild/interpreter/interpreter.py

```diff
@@ -1812,7 +1812,7 @@ def unpack_env_kwarg(self, kwargs: T.Union[build.EnvironmentVariables, T.Dict[st
         msg = ENV_KW.validator(envlist)
         if msg:
             raise InvalidArguments(f'"env": {msg}')
-        return ENV_KW.convertor(envlist)
+        return ENV_KW.converter(envlist)
```

Comment:
This is an object method, seems wrong.

Reply:
Ah, it's been changed everywhere, but this is still changing an (undocumented) API.